RELEASE-NOTES.md

```diff
@@ -1,8 +1,31 @@
-### next
+### v0.6.0
+
+##### Features
+
+- Feature add captcha to registration view ([#317](https://github.com/Code-Poets/sheetstorm/pull/317))
+- Feature add cron job that automatically renew Let's Encrypt Certificate ([#324](https://github.com/Code-Poets/sheetstorm/pull/324))
+- Feature add encryption to postgres backups ([#339](https://github.com/Code-Poets/sheetstorm/pull/339))
+- Feature add modsecurity lib to the nginx ([#320](https://github.com/Code-Poets/sheetstorm/pull/320))
+- Feature add possibility to export monthly reports for every user ([#332](https://github.com/Code-Poets/sheetstorm/pull/332))
+- Feature add signup email confirmation ([#308](https://github.com/Code-Poets/sheetstorm/pull/308))
+- Feature better export of reports ([#331](https://github.com/Code-Poets/sheetstorm/pull/331))
+- Feature mitigating DDoS attack with nginx ([#297](https://github.com/Code-Poets/sheetstorm/pull/297))
+- Feature rebuild register interface ([#283](https://github.com/Code-Poets/sheetstorm/pull/283))
 
 ##### Bugfixes
-- Update `Django` to `2.2.2` to mitigate a security issue
+
+- Bugfix add a missing file that allows you to reset database in the virtual machine ([#338](https://github.com/Code-Poets/sheetstorm/pull/338))
```

Comment:
not sure about this one, please see comment below

Reply:
Why? Virtual machine was added in last release and reseting database was missing in previous configuration

Reply:
OK